### PR TITLE
QF-3108 Always import from django.conf instead os.environ

### PR DESCRIPTION
**Pull request by dddpt**
Affected env variables:
ENVIRONMENT
WEB_HTTPS_PORT
GEODB_HOST
GEODB_PORT
GEODB_DB
GEODB_USER
GEODB_PASSWORD
QFIELDCLOUD_HOST
QFIELDCLOUD_QGIS_IMAGE_NAME
QFIELDCLOUD_WORKER_QFIELDCLOUD_URL
TRANSFORMATION_GRIDS_VOLUME_NAME
QFIELDCLOUD_DEFAULT_NETWORK

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-3108 Always import from django.conf instead os.environ](https://app.clickup.com/t/2192114/QF-3108)

File: docker-app/qfieldcloud/settings.py [hunk 1]
```diff
@@ -180,6 +182,13 @@
     }
 }
 
+# Connexion details for the geodb
```
Comment (suricactus) [inline docker-app/qfieldcloud/settings.py:185]:
```suggestion
# Connection details for the geodb
```

File: docker-app/qfieldcloud/settings.py [hunk 2]
```diff
@@ -49,6 +49,8 @@
 # For example: 'DJANGO_ALLOWED_HOSTS=localhost 127.0.0.1 [::1]'
 ALLOWED_HOSTS = os.environ.get("DJANGO_ALLOWED_HOSTS", "").split(" ")
 
+WEB_HTTPS_PORT = os.environ.get("WEB_HTTPS_PORT")
```
Comment (suricactus) [inline docker-app/qfieldcloud/settings.py:52]:
```suggestion
WEB_HTTP_PORT = os.environ.get("WEB_HTTP_PORT")
WEB_HTTPS_PORT = os.environ.get("WEB_HTTPS_PORT")
```

Just for the sake of completeness we can have both.

File: docker-app/worker_wrapper/wrapper.py [hunk 3]
```diff
@@ -45,8 +44,8 @@
 RETRY_COUNT = 5
 TIMEOUT_ERROR_EXIT_CODE = -1
 DOCKER_SIGKILL_EXIT_CODE = 137
-QFIELDCLOUD_QGIS_IMAGE_NAME = os.environ.get("QFIELDCLOUD_QGIS_IMAGE_NAME", None)
-QFIELDCLOUD_HOST = os.environ.get("QFIELDCLOUD_HOST", None)
+QFIELDCLOUD_QGIS_IMAGE_NAME = settings.QFIELDCLOUD_QGIS_IMAGE_NAME
+QFIELDCLOUD_HOST = settings.QFIELDCLOUD_HOST
```
Comment (suricactus) [inline docker-app/worker_wrapper/wrapper.py:48]:
```suggestion
```

I think these can easily be retired as variables, all can be used is directly `settings.QFIELDCLOUD....`

**Review by suricactus — COMMENTED** [review]
Thanks for the PR. A few comments to be addressed before final review.

File: docker-app/worker_wrapper/wrapper.py (hunk 3 above)
Comment (suricactus) [reply docker-app/worker_wrapper/wrapper.py:48]: As we discussed, it is required down there, but can be replaced with `settings.QFIELDCLOUD...`. Same goes for assert 3 lines below.

**Review by suricactus — APPROVED** [review]
Looks good to me!

**Review by suricactus — APPROVED** [review]

File: docker-app/qfieldcloud/core/tests/test_file_upload_policy.py [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import logging
-import os
 
 import psycopg2
+from django.conf import settings
```
Comment (suricactus) [inline docker-app/qfieldcloud/core/tests/test_file_upload_policy.py:4]:
```suggestion
from django.conf import settings1
```

File: docker-app/qfieldcloud/core/tests/test_file_upload_policy.py [hunk 5]
```diff
@@ -70,16 +70,16 @@ def setUp(self):
         self.geodb = Geodb.objects.create(
             user=self.user,
             dbname="test",
-            hostname="geodb",
-            port=5432,
+            hostname=settings.GEODB_HOST,
```
Comment (suricactus) [inline docker-app/qfieldcloud/core/tests/test_file_upload_policy.py:73]:
```suggestion
            hostname=settings.GEODB_HOST,1
```

**Comment by suricactus** [comment]
@dddpt lint and force push again.

**Pull request merged**